new-frontiers-14/frontier-station-14
# Belching

## About the PR
Adds a Belching/Burping emote to Dwarves, Humans, Oni, & Goblins.

## Why / Balance
Its just a small emote I would like to see in game. If you think it would fit other races it can be easily modified to allow them to, I just didn't know if other species except the near-humans would physiologically be able to.

## How to test
Open the game, play a Human, Dwarf, Oni, or Goblin. Open your Emote Wheel and select Vocal, and then burp away. You can test to see if other races cant by playing other races and attempting the same actions, or manually trying to emote by typing "belch", "burp", "burps", etc. 

## Media
https://youtu.be/LV5-bMR2LXI


## Breaking changes

**Changelog**
- add: A New Belch Emote for Human, Dwarf, Goblin, Oni, and Harpy.
